doc/ref/modules/all/salt.modules.keystone.rst

```diff
@@ -1,6 +0,0 @@
-=====================
```

Comment:
As I can see the keystone module was deprecated and have to be removed but I don't see removal of the module in this PR.

Reply:
That's been punted to the next release - the state was not deprecated, and hasn't been re-written to take advantage of the `keystoneng` module. Looks like I need to put the docs back, though :stuck_out_tongue_closed_eyes: